### DISCUSSION TITLE
How only Rectangle Stroke listen PointerPressedEvent method？

### DISCUSSION THREAD

**Discussion by sparrow2015** [AvaloniaUI/Avalonia#14800] [Q&A] [answered]
I add a Rectangle object on Canvas,  and Add Handler, but `PaintShapePointerPressed` listen the whole rectangle area. The code below:
`newRectangle.AddHandler(PointerPressedEvent, this.PaintShapePointerPressed);`

Now I  want only the rectangle stroke(border) listen `PaintShapePointerPressed` method, inner area of Rectangle listen nothing. Is there a solution？

**Comment by timunie** [comment, marked as answer]
Set Fill to Transparent or any other Brush. By default it is null which cannot be hit.

**Reply by sparrow2015** [reply]
Yes, it works after I delete `newRectangle.Fill =  ` code, thanks very much.
